### PR TITLE
add a fsynth_gain config key to fine tune the FluidSynth output level

### PR DESCRIPTION
**Pull request by fabiangreffrath**
Fixes #https://github.com/fabiangreffrath/crispy-doom/issues/1087

### CODE REVIEW

File: src/m_config.c [hunk 1]
```diff
@@ -1021,6 +1021,13 @@ static default_t extra_defaults_list[] =
 
     CONFIG_VARIABLE_FLOAT(fsynth_reverb_width),
 
+    //!
+    // Fine tune the FluidSynth output level. Default is 1.0,
+    // range is 0.1 - 10.0.
```
Comment (turol) [inline src/m_config.c:1026]: Why this specific range?

**Review by turol — APPROVED** [review]

**Comment by turol** [comment]
Do you intend to add something to setup to adjust this?

File: src/m_config.c (hunk 1 above)
Comment (fabiangreffrath) [reply src/m_config.c:1026]: It's the same range we use in Woof, but it's arbitrary. I'm pretty sure `10.0` will seriously harm your ears, though. Maybe we should limit it to the `0.5..10.0` range.

**Comment by fabiangreffrath** [comment]
> Do you intend to add something to setup to adjust this?

No, it's a rather exotic setting. Most users will just stick to the default volume the soundfont provides. It'd be good to know for users who *really* care that this setting exists at all.

File: src/m_config.c (hunk 1 above)
Comment (rfomin) [reply src/m_config.c:1026]: `10` is a maximum value for [fluid_synth_set_gain()](https://www.fluidsynth.org/api/group__synthesis__params.html#gab1929557284fdfeb59abf63bde83d35c) function.

Comment (turol) [reply src/m_config.c:1026]: That method goes down to zero. Should we allow that? Or some really small number?

Comment (rfomin) [reply src/m_config.c:1026]: We probably should, very loud soundfonts exist too. I think `0` is simpler and more intuitive.

**Comment by fabiangreffrath** [comment]
The subject line of this PR could suffice as a NEWS entry.

**Pull request merged**